File: dk.danskebank.blackbox.client.scala/src/dk/danskebank/markets/trading/client/scala/ClientSample.scala
package dk.danskebank.markets.trading.client.scala

import dk.danskebank.blackbox.client.IClientContract
import dk.danskebank.blackbox.client.TradeAction

class ClientSample extends IClientContract {
  def getStrategyName(): String = ???

  def tick(x$1: Double): TradeAction = ???

}

File: dk.danskebank.blackbox.client.java/src/dk/danskebank/markets/trading/client/ClientSample.java
package dk.danskebank.markets.trading.client;

import dk.danskebank.blackbox.client.IClientContract;
import dk.danskebank.blackbox.client.TradeAction;

public class ClientSample implements IClientContract {

	@Override 
	public String getStrategyName() {
		return "Sample"; 
	} 
  
	@Override 
	public TradeAction tick(double price) {
		return TradeAction.BUY;   
	}
} 